ardent/distribution.yaml

```diff
@@ -867,6 +867,22 @@ repositories:
       url: https://github.com/ros2/urdfdom_headers.git
       version: ros2
     status: maintained
+  vision_msgs:
+    doc:
+      type: git
+      url: https://github.com/Kukanani/vision_msgs.git
+      version: ros2
+    release:
+      tags:
+        release: release/ardent/{package}/{version}
+      url: https://github.com/Kukanani/vision_msgs-release.git
+      version: 1.0.0-0
+    source:
+      test_pull_requests: true
```

Comment:
the ROS 2 buildfarm doesn't have PR or devel jobs at the moment. I think this will just be ignored but can you please remove this line just in case ? thanks!

Reply:
Took care of it, thanks!